x-pack/plugins/observability/public/hooks/slo/use_fetch_group_by_cardinality.ts

```diff
@@ -40,10 +48,16 @@ export function useFetchGroupByCardinality(
                     filter: [{ range: { [timestampField]: { gte: 'now-24h' } } }],
                   },
                 },
+                runtime_mappings: {
+                  group_combinations: {
+                    type: 'keyword',
+                    script: buildInstanceId(groupBy),
+                  },
+                },
```

Comment:
i wonder if it makes sense to use runtime_mappings only when there are multiple group bys?

Reply:
I like keeping the logic consistent and opting for less complex, more maintainable code.